architecture/additional_concepts/other_api_objects.adoc

```diff
@@ -267,6 +267,16 @@ This determines the user's identity, and provides that information to
 
 {product-title} then looks for a `*UserIdentityMapping*` for that `*Identity*`:
 
+ifdef::openshift-enterprise,openshift-origin[]
+[NOTE]
+====
+If the identity provider is configured with the `lookup` mapping method, for example,
+if you are using an external LDAP system, this automatic mapping is not performed. 
+You must create the mapping manually. For more information, 
+see xref:../../install_config/configuring_authentication#LookupMappingMethod [Lookup Mapping Method].
```

Comment:
@mburke5678 - extra space between where the anchor tag finishes and the '['. This makes the link invalid. I have fixed it in master + this branch.